### PR TITLE
[script] [inventory-manager] Inventory manager add storage box

### PR DESCRIPTION
**Pull request by MahtraDR**
Adding as a draft for now, since it's something I was asked for, but that I can't test for myself. Waiting on testing results.

### CODE REVIEW

File: inventory-manager.lic [hunk 1]
```diff
@@ -179,7 +181,41 @@ class InventoryManager
...
+        @item_data[@active_character] << item.lstrip.concat(' (caravan_box)')
+      else
+        item = item.split("and")
+                   .each do |item2| 
+                     @item_data[@active_character] << item2.strip.concat(" (caravan_box)").tr('.', '')
+                   end
+      end
+    end
+
+    DRC.message("Saving caravan box data for #{@active_character}!")
```
Comment (rpherbig) [inline inventory-manager.lic:218]: This says 'caravan box', not 'storage box'. Are the two the same thing?

File: inventory-manager.lic [hunk 2]
```diff
@@ -49,12 +51,12 @@ class InventoryManager
...
 	  elsif args.remove
       remove_character_data(args.name)
-	  elsif args.search
+    elsif args.search
       search_for_item(args.item)
     elsif args.list
       list_character_inv(args.name)
     else
-			DRC.message 'Type ;inventory-manager help for a usage guide'
+      DRC.message("Type ;inventory-manager help for a usage guide")
```
[the remark this answers is not in the document]
Comment (MahtraDR) [reply inventory-manager.lic]: Thank you, good catch here too!

File: inventory-manager.lic (hunk 1 above)
Comment (MahtraDR) [reply inventory-manager.lic:218]: Same thing. The storage box lives on a trader's caravan.

**Review by rpherbig — APPROVED** [review]

**Comment by GreyhallowDR** [comment]
Hey thanks, I tested on mine and it is working as expected now. Thanks!

**Pull request merged**